pkg/kubefedctl/federate/util.go

```diff
@@ -18,25 +18,21 @@ package federate
 
 import (
 	"bufio"
-	"io"
-	"os"
-	"strings"
-
 	"github.com/pkg/errors"
-
+	"io"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
 	versionhelper "k8s.io/apimachinery/pkg/version"
 	"k8s.io/client-go/rest"
-	"sigs.k8s.io/yaml"
-
+	"os"
 	"sigs.k8s.io/kubefed/pkg/apis/core/typeconfig"
 	ctlutil "sigs.k8s.io/kubefed/pkg/controller/util"
 	"sigs.k8s.io/kubefed/pkg/kubefedctl/enable"
-	"sigs.k8s.io/kubefed/pkg/kubefedctl/options"
+	"sigs.k8s.io/kubefed/pkg/kubefedctl/util"
+	"sigs.k8s.io/yaml"
```

Comment:
For better readability, it is import convention in kubefed:
```
import(
    "core-packages"

    "3rd-party-packages"

    "kubefed-packages"
)
```

E.g.:
```
import (
	"io"
	"os"

    	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
	"k8s.io/apimachinery/pkg/runtime/schema"

	"sigs.k8s.io/kubefed/pkg/kubefedctl/enable"
	"sigs.k8s.io/kubefed/pkg/kubefedctl/options"
)
```

Reply:
Fixed it, 
looks like kubefed uses a slightly different import convention:
```
import(
    "core-packages"

    "3rd-party-packages"

    "K8s packages"

    "kubefed-packages"
) ```
Is it correct ?